templates/OperationResponseFailureBodySnippet.stencil

```diff
@@ -0,0 +1,18 @@
+guard let data = httpResponse?.data else {
+    let noDataError = AzureError.sdk("Response data expected but not found.")
+    DispatchQueue.main.async {
+        completionHandler(.failure(noDataError), httpResponse)
+    }
```

Comment:
This will need to be updated based on the changes to master to use the local `dispatchQueue`.